### PR TITLE
MIFOSX-2184 - Add ability to disable backdating of penalties when add…

### PR DESCRIPTION
**Pull request by sheyden**
…ing to a product

### CODE REVIEW

File: mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java [hunk 1]
```diff
@@ -2387,7 +2387,6 @@ private void checkClientOrGroupActive(final Loan loan) {
         }
     }
 
-    @Override
```
Comment (pramodn02) [inline mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java]: why '@Override' removed from here?

File: mifosng-provider/src/main/java/org/mifosplatform/infrastructure/configuration/domain/ConfigurationDomainService.java [hunk 2]
```diff
@@ -49,4 +49,5 @@
 
     int getRoundingMode();
 
+    boolean isBackdatePenalties();
```
Comment (pramodn02) [inline mifosng-provider/src/main/java/org/mifosplatform/infrastructure/configuration/domain/ConfigurationDomainService.java]: please give a proper name for this method (like allowTransactionsOnNonWorkingDayEnabled) and  DB column name

**Comment by rajuan** [comment]
@sheyden Please address the review comments

**Comment by rajuan** [comment]
Please resend with comments addressed

**Pull request closed without merging**